packages/swing-store/src/snapStore.js

```diff
@@ -109,31 +137,38 @@ export function makeSnapStore(
     });
   };
 
-  /** @param {string} hash */
-  function baseNameFromHash(hash) {
-    assert.typeof(hash, 'string');
-    assert(!hash.includes('/'));
-    return `${hash}.gz`;
-  }
+  const sqlDeleteAllUnusedSnapshots = db.prepare(`
```

Comment:
nit: let's move this statement+function to the end, so the definition order matches the order of properties in the returned API object